.github/workflows/linux.yml

```diff
@@ -1,6 +1,27 @@
 name: CI - Linux via APT
 
-on: [push,pull_request]
+on:
+  push:
+    paths-ignore:
+      - 'doc/**'
+      - '.gitlab-ci.yml'
+      - '.gitignore'
+      - '*.md'
+      - 'CITATION.*'
+      - 'COPYING.LESSER'
+      - 'colcon.xml'
```

Comment:
I do not find a file called like this, just `colcon.pkg` .

Reply:
Fixed